runbot_build_instructions/runbot_build.py

```diff
@@ -69,6 +69,27 @@ def job_00_init(self, cr, uid, build, lock_path, log_path):
         build.prebuilt = True
         return res
 
+    def job_10_test_base(self, cr, uid, build, lock_path, log_path):
+        if build.branch_id.repo_id.skip_test_jobs:
+            _logger.info('skipping job_10_test_base')
+            return -2
```

Comment:
Please put a note explaining that this is the expected number in runbot for directly calling next job